src/WarcraftLegacies.Source/Regions.cs

```diff
@@ -212,6 +212,7 @@ public static class Regions
 	public static Rectangle MonolithNoBuild { get; set; } = new Rectangle(18208f, -31584f, 23808f, -23360f);
 	public static Rectangle Moonglade { get; set; } = new Rectangle(-12992f, 10176f, -12416f, 10656f);
 	public static Rectangle MoongladeVillage { get; set; } = new Rectangle(-14688f, 10048f, -12288f, 12192f);
+	public static Rectangle MountHyjal { get; set; } = new Rectangle(-11328f, 9856f, -8832f, 11264f);
```

Comment:
This change shouldn't be here. Could you please pull `dev` into your branch and push? Should fix it.